test/pdfs/.gitignore

```diff
@@ -314,6 +314,7 @@
 !issue10339_reduced.pdf
 !issue4934.pdf
 !issue4650.pdf
+!issue13132.pdf
```

Comment:
Also, given that this isn't a reduced test-case but a full PDF document (with unclear copyright situation), this should probably be a *linked* test-case instead.